docs/medair.md

```diff
@@ -0,0 +1,70 @@
+# nf-core/configs: Medair Configuration
+
+All nf-core pipelines have been successfully configured for use on the Medair cluster at Clinical Genomics Gothenburg.
+
+To use, run the pipeline with `-profile medair`. This will download and launch the [`medair.config`](../conf/medair.config) which has been pre-configured with a setup suitable for the Medair cluster.
+It will enable Nextflow to manage the pipeline jobs via the `SGE` job scheduler.
+Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+You will need an account to use the Medair cluster in order to download or run pipelines. If in doubt, contact cgg-it.
+
+## Download nf-core pipelines
+
+### Set-up: load Nextflow and nf-core tools
+
+First you need to load relevant softwares: Nextflow and nf-core tools. You can do it as follow:
+
+```bash
+## Load Nextflow
+module load nextflow
+## Load nf-core tools
+module load miniconda
+source activate nf-core
+```
+
+### Storage of Singularity images
+
+When downloading a nf-core pipeline for the first time (or a specific version of a pipeline), you can choose to store the Singularity image for future use. A central location for these images is: `/apps/bio/dependencies/nf-core/singularities`.
+
+If you run or add the following to your `.bashrc`, Nexflow will know where to store the images:
+
+```bash
+export NXF_SINGULARITY_CACHEDIR="/apps/bio/dependencies/nf-core/singularities"
```

Comment:
This is a little confusing, by your config above says it'll always go to the `singularities` folder? A user doesn't ave an option in this case.

Reply:
True. I clarified it now.